src/machine/usb/hid/keyboard/keyboard.go

```diff
@@ -63,8 +63,14 @@ func init() {
 	}
 }
 
-// New returns hid-keybord.
+// New returns the USB hid-keyboard port.
+// Deprecated, better to just use Port()
```

Comment:
If the deprecation comment starts a new paragraph and uses a colon, it will be flagged up by standard Go tooling (like staticcheck) at call sites
```
//
// Deprecated: Use Port()
```